test/gtest/ucp/test_ucp_tag_offload.cc

```diff
@@ -329,9 +329,14 @@ class test_ucp_tag_offload_multi : public test_ucp_tag_offload {
     void activate_offload_hashing(entity &se, ucp_tag_t tag)
     {
         se.connect(&receiver(), get_ep_params());
-        // Need to send twice, since the first message may not enable hashing
-        // (num_active_iface on worker is increased after unexpected offload handler)
-        send_recv(se, tag, 2048);
+        // Need to send twice:
+        // 1. to ensure that wireup's UCT iface has been closed and
+        //    it is not considered for num_active_iface on worker
+        //    (message has to be less than `UCX_TM_THRESH` value)
+        // 2. to activate tag oofload
+        //    (num_active_iface on worker is increased after unexpected offload
```

Comment:
num_active_iface is increased when any message is received on any iface. Tag hashing is done when we have more than 1 active ifaces

Reply:
done